bin/node/cli/Cargo.toml

```diff
@@ -156,3 +156,4 @@ runtime-benchmarks = [
 	"node-runtime/runtime-benchmarks",
 	"frame-benchmarking-cli",
 ]
+rhd = ["default"]
```

Comment:
We don't have rhd anywhere in the codebase

Reply:
Yup for some reason I clearly remember having it pop out somewhere... gonna remove this one and keep investigating :)

Reply:
Oh I know why! We previously had a `#[cfg(feature = "rhd")]` here: https://github.com/paritytech/substrate/blob/606b4faf2f52ec588235d3cdff6c7364a393a645/bin/node/cli/src/service.rs#L436 but it has now been removed :)